### PR TITLE
V1.3 add reset command

### PR DESCRIPTION
**Pull request by inezkok**
Add Reset Command (for overtime hours and allocated leaves)

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ResetCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    @Test
+    void execute_missingField_failure() {
+        ResetCommand resetCommand = new ResetCommand(NO_FIELD);
+
+        String expectedMessage = ResetCommand.MESSAGE_NO_FIELD;
+
+        assertCommandFailure(resetCommand, model, expectedMessage);
+    }
+
```
Comment (LordSaumya) [inline src/test/java/seedu/address/logic/commands/ResetCommandTest.java:84]: Tests look good. Can maybe also test if resetting an empty or filtered list does anything.

File: src/test/java/seedu/address/logic/parser/ResetCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+public class ResetCommandParserTest {
+    private ResetCommandParser parser = new ResetCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, ResetCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsSortCommand() {
```
Comment (LordSaumya) [inline src/test/java/seedu/address/logic/parser/ResetCommandParserTest.java:21]: `returnsSortCommand` and `expectedSortCommand` need to be changed to `returnsResetCommand` and `expectedResetCommand`.

**Review by LordSaumya — COMMENTED** [review]
Looks good, minor changes required.

**Review by LordSaumya — APPROVED** [review]
